components/EventsList.tsx

```diff
@@ -24,11 +25,15 @@ const EventsList = () => {
   }, [])
 
   // variables
-  const snapPoints = ["4%", "65%", "100%"]
+  const snapPoints = ["3%", "65%", "100%"]
```

Comment:
We'll probably want the min height to be more like 10% (so that the user can see the "Nearby Activities" text) and the last snap point to be more like 75% (100% makes the sheet practically unusable on iOS)